project/Http4sPlugin.scala

```diff
@@ -53,7 +87,117 @@ object Http4sPlugin extends AutoPlugin {
     mimaFailOnProblem := http4sMimaVersion.value.isDefined,
     mimaPreviousArtifacts := (http4sMimaVersion.value map {
       organization.value % s"${moduleName.value}_${scalaBinaryVersion.value}" % _
-    }).toSet
+    }).toSet,
+
+
+    http4sBuildData := {
+      val dest = target.value / "hugo-data" / "build.toml"
+      val (major, minor) = http4sApiVersion.value
+
+      val releases = latestPerMinorVersion(baseDirectory.value)
+        .map { case ((major, minor), v) => s""""$major.$minor" = "${v.string}""""}
+        .mkString("\n")
+
+      // Would be more elegant if `[versions.http4s]` was nested, but then
+      // the index lookups in `shortcodes/version.html` get complicated.
+      val buildData: String =
+        s"""
+           |[versions]
+           |"http4s.api" = "$major.$minor"
+           |"http4s.current" = "${version.value}"
+           |"http4s.doc" = "${docExampleVersion(version.value)}"
+           |scalaz = "${scalazVersion.value}"
+           |circe = "${circeJawn.revision}"
+           |cryptobits = "${cryptobits.revision}"
+           |"argonaut-shapeless_6.2" = "1.2.0-M5"
```

Comment:
Why is this hard-coded?

Reply:
I think the main build doesn't depend on it, but it also doesn't depend on cryptobits.  This is not new, but it is curious.

Reply:
This one is not actually compiled into the tuts.  Unless it ever is, adding it as a dependency would just download another piece of the internet.